plugins/wdc/wdc-nvme.c

```diff
@@ -4021,17 +4021,12 @@ static void wdc_print_fw_act_history_log_normal(__u8 *data, int num_entries, __u
 	char *null_fw = "--------";
 	memset((void *)time_str, 0, 11);
 
+	printf("  Firmware Activate History Log \n");
+	printf("           Power on Hour       Power Cycle     Previous    New                               \n");
+	printf("  Entry      hh:mm:ss             Count        Firmware    Firmware    Slot   Action  Result \n");
+	printf("  -----  -----------------  -----------------  ---------   ---------   -----  ------  -------\n");
+
 	if (data[0] == WDC_NVME_GET_FW_ACT_HISTORY_C2_LOG_ID) {
-		printf("  Firmware Activate History Log \n");
-		if (cust_id == WDC_CUSTOMER_ID_0x1005) {
```

Comment:
There are certain FW versions that return Power On Hour in the hh:mm:ss format and others that return it in a timestamp format.  So I believe the cust_id check needs to remain here.

Reply:
Addressed.